<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassicTagGameWindow</class>
 <widget class="QDialog" name="ClassicTagGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassicTagGameWindow</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">        QPushButton {
            background-color: lightgray;
            border: 1px solid gray;
            padding: 5px;
        }

        QPushButton:pressed {
            background-color: darkgray;
            border: 1px solid black;
        }

        QPushButton:hover {
            background-color: gainsboro;
        }

        QPushButton:disabled {
            background-color: #ddd;
            color: #888;
            border: 1px solid #bbb;
        }
</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>67</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_shuffle">
      <property name="text">
       <string>Перемешать поле</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="text">
       <string>Выйти в меню</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
